.bettercodehub.yml

```diff
@@ -0,0 +1,19 @@
+component_depth: 1
```

Comment:
```suggestion
component_depth: 1
default_excludes: true
```

Is there a reason not to include the default excludes?

Reply:
Where did you find `default_excludes`? It's not in [their configuration documentation](https://bettercodehub.com/docs/configuration-manual). I can imagine that it might be helpful, but I don't know what it does.

Reply:
It was in the default config of this repo.

Reply:
I'm lost. Which repo? There was no default BetterCodeHub config for the 3601-iteration-template repo, and I can't find any evidence that BetterCodeHub supports a `default_excludes` entry in their config file.

Reply:
If you go to the bettercodehub site and open the settings for the repo it shows the current config.

Reply:
Yup. I totally see it now. I have no idea why that's not in their docs on the form of these files. I'll definitely add that.